build/assert-publish-aot-testApp-warning-count.ps1

```diff
@@ -12,6 +12,20 @@ foreach ($line in $($publishOutput -split "`r`n"))
     }
 }
 
+pushd $rootDirectory/test/OpenTelemetry.AotCompatibility.TestApp/bin/Debug/net7.0/linux-x64

```

Comment:
This is good for now, later we might need to find a way to remove the hard-coded "net7.0" version (by taking the version from environment variables).

Reply:
For example, https://github.com/open-telemetry/opentelemetry-dotnet/blob/a831093ae18efeb7bcbcef1042d05c0cef05e7a8/.github/workflows/ci-instrumentation-libraries.yml#L41